vercel/resource_project.go

```diff
@@ -206,11 +210,48 @@ At this time you cannot use a Vercel Project resource with in-line ` + "`environ
 							stringLengthBetween(1, 72),
 						},
 					},
-					"protect_production": schema.BoolAttribute{
-						Description: "If true, production deployments will also be protected",
-						Optional:    true,
-						Computed:    true,
-						Default:     booldefault.StaticBool(false),
+					"deployment_type": schema.StringAttribute{
+						Required:      true,
+						Description:   "The deployment environment to protect. Must be one of `standard_protection`, `all_deployments`, or `only_preview_deployments`.",
+						PlanModifiers: []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+						Validators: []validator.String{
+							stringOneOf("standard_protection", "all_deployments", "only_preview_deployments"),
+						},
+					},
+				},
+			},
+			"trusted_ips": schema.SingleNestedAttribute{
+				Description: "Ensures only visitors from an allowed IP address can access your deployment.",
+				Optional:    true,
+				Attributes: map[string]schema.Attribute{
+					"addresses": schema.SetAttribute{
+						Description:   "The allowed IP addressses and CIDR ranges with optional descriptions.",
+						Required:      true,
+						PlanModifiers: []planmodifier.Set{setplanmodifier.UseStateForUnknown()},
+						ElementType: types.ObjectType{
+							AttrTypes: map[string]attr.Type{
```

Comment:
I can't seem to remove addresses. Could be how the API is consumed?

Reply:
edit: never mind, it's fine. 😖

Reply:
edit edit: there is something weird going on here. 
If I add an `address` in the UI, `terraform plan` suggests it will be deleted, but `terraform apply` makes no changes.

Reply:
It worked as expected for me. I added an IP in the UI, then ran terraform apply without the value and it was deleted

Reply:
Interesting.

Reply:
Right, yeah, it's my fault. I have `-refresh=false` set in the alias I usually run `terraform apply` with.